lmnet/configs/core/classification/lmnet_v1_quantize_cifar10.py

```diff
@@ -91,6 +91,7 @@
 }
 NETWORK.WEIGHT_QUANTIZER = binary_mean_scaling_quantizer
 NETWORK.WEIGHT_QUANTIZER_KWARGS = {}
+NETWORK.QUANTIZE_FIRST_CONVOLUTION = True
```

Comment:
Is `QUANTIZE_FIRST_CONVOLUTION` needs ?

Reply:
This one is not needed so I'm gonna delete it in the next commit.

Reply:
The code below is also not necessary.
Thank you so much for pointing it out.